src/screens/FeedScreen/elements/AddCommentPreview.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {TouchableOpacity, StatusBar} from 'react-native';
+import WriteComment from '../../../components/Comments/WriteComment';
+import {COLORS} from '../../../utils/theme';
+import {normalize} from '../../../utils/fonts';
+
+type AddCommentPreviewProps = {
+  username: string;
+  isBlurred: boolean;
+  heightReaction: number;
+  onPressComment: () => void;
+};
+
+const tabBarHeight = StatusBar.currentHeight;
+
+function AddCommentPreview({
+  username,
+  isBlurred,
+  heightReaction,
+  onPressComment
+}: AddCommentPreviewProps) {
+  return (
+    !isBlurred && (
+      <TouchableOpacity
+        testID="writeComment"
+        onPress={onPressComment}
+        style={{
+          maxHeight: heightReaction,
+          marginBottom: heightReaction <= 0 ? tabBarHeight + normalize(10) : 0,
+          borderTopWidth: 1,
+          borderTopColor: COLORS.light_silver
+        }}>
+        <WriteComment
+          postId={''}
+          username={username}
+          value={null}
+          loadingPost={false}
+          isViewOnly={true}
+          withAnonymityLabel={false}
+          onChangeText={() => {
+            return null;
+          }}
+          onPress={() => {
+            return null;
+          }}
+        />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `WriteComment` component is being used with hardcoded values. This could lead to issues if these properties are required or expected to change. If these properties are not necessary, consider creating a new simpler component that doesn't require unnecessary props.

```diff
-         <WriteComment
-           postId={''}
-           username={username}
-           value={null}
-           loadingPost={false}
-           isViewOnly={true}
-           withAnonymityLabel={false}
-           onChangeText={() => {
-             return null;
-           }}
-           onPress={() => {
-             return null;
-           }}
-         />
+         <SimpleWriteComment
+           username={username}
+         />
```
In this case, `SimpleWriteComment` would be a new component that only takes the necessary props.


<!-- This is an auto-generated comment by OSS CodeRabbit -->